srtcore/core.cpp

```diff
@@ -9323,7 +9359,7 @@ void CUDT::addLossRecord(std::vector<int32_t> &lr, int32_t lo, int32_t hi)
     }
 }
 
-void CUDT::checkACKTimer(const steady_clock::time_point &currtime)
+void CUDT::checkACKTimer(const steady_clock::time_point &currtime, char debug_decision[10])
 {
     if (currtime > m_tsNextACKTime  // ACK time has come
```

Comment:
Better to return an integer related to ACK type

Reply:
I'd have to make bit flags for it, as they can come from various different reasons.